packages/cozy-flags/src/connect.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+
+/**
+ * Connects a component to the flags. The wrapped component
+ * will be refreshed when a flag changes.
+ */
+const connect = Component => {
+  class Wrapped extends React.Component {
+    constructor(props) {
+      super(props)
+      this.handleChange = this.handleChange.bind(this)
+    }
+    componentWillMount() {
```

Comment:
Can we use `didMount` instead? Since `willMount` is a legacy API

Reply:
Thanks, done in https://github.com/cozy/cozy-libs/pull/453/commits/30c2e7845ca49aac59cb0d7dcbc76831a06dd64a